### ISSUE TITLE
Feature request: Add "no button" option for cards and jumbotrons

### ISSUE THREAD

**Issue by tjsuw** [uweb/uw_wp_theme#8]
Request from @jh1uw :

Add a flag to the card and jumbotron shortcodes to turn off the button

**Comment by lcaple** [comment]
from Jill @ Pyramid:

We've found the requirement to have a button on all cards (except block), and to disallow buttons on block cards challenging. As we started experimenting with different page layouts, we found ourselves really leaning on cards to highlight content that either didn’t have a call to action (purely informational), or contained a list of multiple links, rather than a single CTA. Forcing the button made it challenging to use them in both situations. Given how robust the styling options are, we were wondering if having more options around the use of a button might be helpful for editors.

**Comment by lcaple** [comment]
From Jill @ Pyramid:

We found several occasions where we wanted to use the Jumbotrons, but wound up finding other solutions because of the requirement to have a button. This was particularly true for the use case of having it as a page header/hero — we found ourselves just wanting to display the department’s name or name + intro text, but without a CTA. Wondering if there might be value in making the button the default, but optional.
